src/js/cptui-scripts.js

```diff
@@ -180,15 +276,15 @@ postboxes.add_postbox_toggles(pagenow);
 		}).join("");
 	}
 
-	if ( undefined != wp.media ) {
-		var _custom_media = true,
-			_orig_send_attachment = wp.media.editor.send.attachment;
+	if ( undefined !== wp.media ) {
+		_custom_media = true;
+		_orig_send_attachment = wp.media.editor.send.attachment;
```

Comment:
This line also has risk for errors, consider using destruction instead. (Just a suggestion)

Reply:
Going to pass on this still valid suggestion for the time being.